common/supplemental/coverageLevels.xml

```diff
@@ -433,11 +433,11 @@ For terms of use, see http://www.unicode.org/copyright.html
 		<coverageLevel value="basic" match="listPatterns/listPattern/listPatternPart[@type='(2|start|middle|end)']"/>
 		<coverageLevel value="basic" match="units/durationUnit[@type='(hm|ms|hms)']/durationUnitPattern"/>
 		<coverageLevel value="moderate" match="localeDisplayNames/languages/language[@type='%language60']"/>
-		
+
```

Comment:
spurious changes, probably tabs

Reply:
should I bring back those tabs? Or I can leave them removed, as the rest of the file. I think we can go ahead.

Reply:
leave them